src/controllers/confirm.company.controller.ts

```diff
@@ -27,12 +32,24 @@ const INELIGIBLE_PAGES = {
  * @param next
  */
 
-export const get = (req: Request, res: Response, next: NextFunction) => {
+export const get = async (req: Request, res: Response, next: NextFunction) => {
   if (req.session) {
     try {
       const company: ObjectionCompanyProfile = retrieveCompanyProfileFromObjectionSession(req.session);
+
+      const session: Session = req.session as Session;
```

Comment:
could this be moved to the top of the get and used in the if?

Reply:
I think I see what you mean - move it up and use in the call to retrieve the company profile? Will push a change.